api-tests/specs/practitioner-services/createperson.js

```diff
@@ -0,0 +1,37 @@
+const { validPerson, invalidCharacterLength, missingGivenName, missingfamilyName, missingPrefix } = require('../../data/createPerson')
+const conf = require('../../config/conf')
+
+beforeEach(function () {
+    baseRequest = request(conf.BASE_URL)
+})
+
+describe('As a user with Create Person permission, I want to have my create person request validated by the system, So that I cannot create an invalid person', function () {
```

Comment:
Some more cases we can test for:
- Illegal characters (see the regex)
- Unresponsive (not existing) fhir store 
- Malformed input

Reply:
Hi @balteravishay I have added the illegal characters check scenario, not sure how to cover the unresponsive (FHIR) store, please advise. Also I have already covered malformed input as part of other scenarios.

Reply:
With regards to the 'Malformed input' the invalid json format is not acceptable and replicating an unresponsive FHIR store would just show a 404 which I am not sure would add any value to the tests.

Reply:
Hi @balteravishay the malformed script isn't showing a 400 when run using mocha but somehow postman is able to show it. I added the script which can anyways assert it.